seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -721,19 +738,43 @@
         "  // change code below this line  ",
         "",
         "  // change code above this line",
```

Comment:
@QuincyLarson merging this currently would be problematic I think. See what I've noted below. This would splice 2 challenges together (object tests w/ array challenge), and it removes an additional test that was added to that last object challenge.

The content that you added in here has actually been moved up to the top as the first array challenge (it was just posted out of order on the GitHub thread).